src/eduid/satosa/scimapi/scim_attributes.py

```diff
@@ -19,6 +20,7 @@ class Config(object):
     mongo_uri: str
     idp_to_data_owner: Mapping[str, str]
     mfa_stepup_issuer_to_entity_id: Mapping[str, str]
+    virt_idp_to_data_owner: Mapping[str, str]
```

Comment:
Do we expect virtual IdP names to conflict with plain-IdP names?
ie, is there a reason to have a separate option for this, instead of reusing and extending `idp_to_data_owner`?

IMO, the only thing that needs to be defined is the lookup order
- first, lookup the owner based on the virtual-IdP
- then, try the plain-IdP